js/clinicalTimeline.js

```diff
@@ -13,7 +13,7 @@ window.clinicalTimeline = (function(){
       enableZoom = true,
       stackSlack = null,
       translateX = 0,
-      beginning = "0",
+      beginning = 0,
```

Comment:
This breaks on JSON input that starts at day 0. I think it's some problem with the d3-timeline library, or the way we use it.